README.md

```diff
@@ -1,158 +1,274 @@
+
+  <p><strong>Step 1:</strong> Using your Ewon device’s webpage, open the user account settings by selecting <code>Setup</code> &gt; <code>Users</code> in the menu on the left.</p>
+  <img src="images/ewonsetup-step1.png" width="173" />
+  <br />
+
+  <p><strong>Step 2:</strong> Create a new user account with the following information. Do not modify user rights.</p>
+  <p><code>User Login</code>: <em>demo</em><br />
+  <code>Password</code>: <em>demopassword</em><br />
+  <code>Confirm Password</code>: <em>demopassword</em></p>
+  <p><strong>NOTE:</strong> This user account is required for Ewon Ignition Demo to function and allows the demo to configure tags on your Ewon device.</p>
+  <img src="images/ewonsetup-step2.png" width="440" />
+  <br />
+
+  <p><strong>Step 3:</strong> Open an FTP connection to your Ewon device using your operating system’s file manager (i.e. File Explorer).</p>
+  <p>To do this, type the following address in the address bar, replacing “[EWON IP HERE]” with the IP address you use to access your Ewon device’s webpage, then hit Enter/Return. <em>If required, enter the username/password you use to access your Ewon device’s webpage.</em></p>
+  <p><code>Address</code>: <em>ftp://[EWON IP HERE]</em></p>
+  <img src="images/ewonsetup-step3.png" width="600" />
+  <br />
+
+  <p><strong>Step 4:</strong> Using the created FTP connection, copy the files <code>EwonIgnitionDemoApp.jar</code> and <code>jvmrun</code> from the demo resource pack to the <code>usr</code> folder on your Ewon device.</p>
+  <img src="images/ewonsetup-step4.png" width="504" />
+  <br />
+
+  <p><strong>Step 5:</strong> Reboot your Ewon device by disconnecting and reconnecting power, or by selecting <code>Setup</code> &gt; <code>Reboot</code> on your Ewon device’s webpage.</p>
+  <img src="images/ewonsetup-step5.png" width="159" />
+
+  <!-- INSTALLATION/IGNITION SETUP -->
+  <h3 id="ignition-setup"><span style="font-variant: small-caps;">Ignition Setup</span></h3>
+  <p><em>During Ignition Setup, you’ll need access to your Ignition gateway’s webpage. For information on installing and accessing Ignition, please refer to <a href="#installing-and-configuring-ignition">Appendix 6.4, Installing and Configuring Ignition</a>.</em></p>
+
+  <p><strong>Step 1:</strong> Download and install the latest version of the Ignition Ewon Connector from the IgnitionEwonConnector GitHub repository, found here: <a href="https://github.com/hms-networks/IgnitionEwonConnector" class="uri">https://github.com/hms-networks/IgnitionEwonConnector</a>.</p>
+  <p>For instructions on how to download and install the connector, please refer to the “Download”, “Installation” and “Ewon Setup” sections of the Ignition Ewon Connector GitHub repository.</p>
+  <p><strong>NOTE:</strong> You may omit the “Setup Historical Logging for Tags” step in “Ewon Setup.”</p>
+  <img src="images/ignitionsetup-step1.png" width="400" /></p>
```

Comment:
Why is there a mouse cursor in this screenshot?

Reply:
@TomKimsey What mouse cursor...? 🤔